docs/src/__examples__/Grid/DEFAULT.tsx

```diff
@@ -28,9 +28,51 @@ export default {
       </Grid>
     );
   },
-  info: {
-    title: "Default grid",
-    description:
-      "By default, grids are displayed as block elements and display their columns and rows without any gaps",
-  },
+  exampleKnobs: [
+    {
+      component: "Grid",
+      knobs: [
+        {
+          name: "as",
+          type: "text",
+          defaultValue: "div",
+        },
+        {
+          name: "gap",
+          type: "text",
+          defaultValue: "",
+        },
+        {
+          name: "rows",
+          type: "text",
+          defaultValue: "",
+        },
+        {
+          name: "rowGap",
+          type: "text",
+          defaultValue: "",
```

Comment:
Why is `defaultValue` usually empty? People might often not know the format of the value.